docs/contributor/01-10-installation.md

```diff
@@ -0,0 +1,251 @@
+# Install Application Connector Manager
+
+## Prerequisites
+
+- Access to a Kubernetes (v1.24 or higher) cluster
+- [k3d](https://k3d.io) to get a local cluster for testing, or run on a remote cluster
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kubebuilder](https://book.kubebuilder.io/)
+
+## Install kubebuilder
+
+<div tabs name="Install kubebuilder" group="kubebuilder">
+  <details open>
+  <summary label="Homebrew">
+  Homebrew
+  </summary>
+Run:
+
+   ```bash
+   brew install kubebuilder
+   ```
+  </details>
+  <details>
+  <summary label="kubectl">
+  Fetch sources
+  </summary>
+
+Run:
+
+   ```bash
+   curl -L -o kubebuilder https://go.kubebuilder.io/dl/latest/$(go env GOOS)/$(go env GOARCH)
+   chmod +x kubebuilder && mv kubebuilder /usr/local/bin/
+   ```
+  </details>
+</div>
+
+## Manual installation of Application Connector Manager
+
+1. Clone the project.
+
+   ```bash
+   git clone https://github.com/kyma-project/application-connector-manager.git && cd application-connector-manager/
+   ```
+
+2. Set the Application Connector Manager image name.
+
+   ```bash
+   export IMG=custom-application-connector-manager:0.0.1
+   export K3D_CLUSTER_NAME=application-connector-manager-demo
+   ```
+
+3. Build the project.
+
+   ```bash
+   make build
+   ```
+
+4. Build the image.
+
+   ```bash
+   make docker-build
+   ```
+
+5. Push the image to the registry.
+
+<div tabs name="Push image" group="application-connector-installation">
+  <details>
+  <summary label="k3d">
+  k3d
+  </summary>
+
+   ```bash
+   k3d image import $IMG -c $K3D_CLUSTER_NAME
+   ```
+  </details>
+  <details>
+  <summary label="Docker registry">
+  Globally available Docker registry
+  </summary>
+
+   ```bash
+   make docker-push
+   ```
+
+  </details>
+</div>
+
+6. Deploy Application Connector Manager.
+
+   ```bash
+   make deploy
```

Comment:
We are missing step of adding istio resources to k3d cluster. Without them the operator will not be able to install AC components. It can be done with command:

`make -C ./hack/common patch-with-istio-crds `